### PR TITLE
Do not run agvtool while under test

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane/lib/fastlane/actions/increment_build_number.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def self.run(params)
         # More information about how to set up your project and how it works:
         # https://developer.apple.com/library/ios/qa/qa1827/_index.html
         # Attention: This is NOT the version number - but the build number
-        agv_enabled = system([command_prefix, 'agvtool what-version', command_suffix].join(' '))
+        agv_enabled = system([command_prefix, 'agvtool what-version', command_suffix].join(' ')) unless Helper.test?
```
Comment (taquitos) [inline fastlane/lib/fastlane/actions/increment_build_number.rb]: This is confusing to me, but maybe I'm crazy. `agv_enabled = (something) unless we're in test mode.`
would it make sense to more explicitly say it is disabled instead.

Comment (ohayon) [inline fastlane/lib/fastlane/actions/increment_build_number.rb]: yup it would!

**Review by taquitos — APPROVED** [review]
So much clearer now, 🐕

**Pull request merged**